lib/files.cf

```diff
@@ -1674,6 +1674,9 @@ body perms system_owned(mode)
 
     solaris::
       groups => { "sys" };
+
```

Comment:
Missing **s** in *system* in the commit message.

Reply:
ACK